<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REUsers</class>
 <widget class="QDialog" name="REUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Last name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>67</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>67</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="EQLineEdit" name="leGroupName">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="EQLineEdit" name="leGroupCode">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="Field" stdset="0">
        <string>f_group</string>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="EQLineEdit" name="leCode">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="PrimaryKey" stdset="0">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_id</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="EQLineEdit" name="leFName">
       <property name="Field" stdset="0">
        <string>f_firstName</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="EQLineEdit" name="leLName">
       <property name="Field" stdset="0">
        <string>f_lastName</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="EQLineEdit" name="leUsername">
       <property name="Field" stdset="0">
        <string>f_username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="EQComboBox" name="cbState">
       <property name="Cache" stdset="0">
        <number>41</number>
       </property>
       <property name="Field" stdset="0">
        <string>f_state</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">eqcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leCode</tabstop>
  <tabstop>cbState</tabstop>
  <tabstop>leGroupCode</tabstop>
  <tabstop>leUsername</tabstop>
  <tabstop>leFName</tabstop>
  <tabstop>leLName</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>leGroupName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
